### DISCUSSION TITLE
No autocomplete suggestions when adding certain crates

### DISCUSSION THREAD

**Discussion by dlen** [rust-lang/rust-analyzer#16999] [General]
Hi folks!

Couple of days ago I updated my rust install to the latest stable. 1.77.1 Suddenly rust-analyzer didn't show any more autocomplete suggestions in projects where r-a was working fine. I have been trying to narrow down the issue and managed to get a reproducible minimal project

Cargo.toml:
```
[package]
name = "test-ra"
version = "0.1.0"
edition = "2021"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
google-cloudkms1 = "5.0.4"
google-sqladmin1 = "5.0.4"
google-storage1 = "5.0.4"
```

With that specific Cargo.toml r-a does not autocomplete any code. I have tried with nvim/vscode (in both linux and macOS) with no luck with either of them.

In any other scenario r-a works fine.

Any ideas are welcome.

**Comment by alibektas** [comment]
I will look into this in a couple of hours 👍

**Comment by alibektas** [comment]
So I have looked into this for different versions and everything seems to be working just fine. You mentioned that you tried this even with different OSs so I am not going to tell you that maybe the problem is with your setup.  I don't know, you may want to check your rust-analyzer version and update it if necessary. Other than that I don't have much to say, sorry!

**Reply by dlen** [reply]
Thanks a lot for taking the time to look into it!! I know it sounds weird! Most likely it is my setup but it just started to happen recently and I have been banging my head with the issue. Some of my coworkers can reproduce it too on Linux so I hoped is something reproducible somehow by the rest of the community :disappointed: 

I have added a video of the behavior I'm experiencing just in case I'm doing some blatant mistake. It is using vscode (mostly default install as I don't use it) it happens exactly the same with other editors such as neovim.

Rustc version is 1.77.1
rust-analyzer 1.77.1 (7cf61eb 2024-03-27)

https://github.com/rust-lang/rust-analyzer/assets/1511212/28de05c6-d049-4426-be48-10a44545dc35
